### ISSUE TITLE
form on webauthn page is not a <form>

### ISSUE THREAD

**Issue by nlhkabu** [pypa/warehouse#6472]
On `warehouse/templates/manage/account/webauthn-provision.html` we have a form where users can provision their device.

This is *not* wrapped in `<form>` tags, which means it is not semantic.

**Comment by nlhkabu** [comment]
@woodruffw I have reassigned this one to you - do you think there is a workaround we can find that will not break the functionality?  If not, let's close this ticket.

Thanks :)

**Comment by woodruffw** [comment]
Thanks!

Yeah, I *think* there should be a workaround that includes both the `<form>` and a bit of JS (similar to the bit we reverted in the last PR) to perform the proper handling. I'll take a look today.
